assets/src/edit-story/components/panels/document/publish/publish.js

```diff
@@ -204,63 +156,51 @@ function PublishPanel() {
-          <Row>
-            <FieldLabel id={authorLabelId}>
-              {__('Author', 'web-stories')}
-            </FieldLabel>
-            {isUsersLoading || !visibleOptions ? (
-              <AdvancedDropDown
-                placeholder={__('Loading…', 'web-stories')}
-                disabled
-                primaryOptions={[]}
-                {...dropDownParams}
-              />
-            ) : (
-              <AdvancedDropDown
-                options={queriedUsers}
-                primaryOptions={visibleOptions}
-                searchResultsLabel={__('Search results', 'web-stories')}
-                disabled={isSaving}
-                {...dropDownParams}
-              />
-            )}
-          </Row>
+          <Author />
         )}
-        <Row>
-          {/* @todo Replace this with selection to choose between publisher logos */}
-          <LabelWrapper>
-            <FieldLabel>{__('Publisher logo', 'web-stories')}</FieldLabel>
-            <Required />
-          </LabelWrapper>
-          <MediaWrapper isHighlighted={highlightLogo?.showEffect}>
-            <Media
-              ref={publisherLogoRef}
-              value={publisherLogoUrl}
-              onChange={handleChangePublisherLogo}
-              title={__('Select as publisher logo', 'web-stories')}
-              buttonInsertText={__('Select as publisher logo', 'web-stories')}
-              type={allowedImageMimeTypes}
-              ariaLabel={__('Publisher logo', 'web-stories')}
-              variant={MEDIA_VARIANTS.CIRCLE}
-            />
-          </MediaWrapper>
-        </Row>
         <HighlightRow isHighlighted={highlightPoster?.showEffect}>
-          <LabelWrapper>
-            <FieldLabel>{__('Poster image', 'web-stories')}</FieldLabel>
-            <Required />
-          </LabelWrapper>
-          <MediaWrapper isHighlighted={highlightPoster?.showEffect}>
-            <Media
-              ref={posterButtonRef}
-              value={featuredMedia?.url}
-              onChange={handleChangePoster}
-              title={__('Select as poster image', 'web-stories')}
-              buttonInsertText={__('Select as poster image', 'web-stories')}
-              type={allowedImageMimeTypes}
-              ariaLabel={__('Poster image', 'web-stories')}
-            />
-          </MediaWrapper>
+          <MediaInputWrapper>
+            <MediaWrapper isHighlighted={highlightPoster?.showEffect}>
+              <Sizer width={54} height={96}>
+                <Media
+                  ref={posterButtonRef}
+                  value={featuredMedia?.url}
+                  onChange={handleChangePoster}
+                  title={__('Select as poster image', 'web-stories')}
+                  buttonInsertText={__('Select as poster image', 'web-stories')}
+                  type={allowedImageMimeTypes}
+                  ariaLabel={__('Poster image', 'web-stories')}
+                />
+              </Sizer>
+            </MediaWrapper>
```

Comment:
The highlight rectangle around this one looks a bit off when coming from the checklist. I think it would be better if it was the actual bg element of the media, that had a blue outline (also following border-radius):

![Screen Shot 2021-03-15 at 21 13 43](https://user-images.githubusercontent.com/637548/111241441-84f25600-85d3-11eb-9aa0-f44d42636625.png)